File: Bridge Pattern/PHP/Bmodel.php
<?php

require_once "Model.php";
require_once "Type.php";

class Bmodel extends Model 
{
    private Type $type;

    public function __construct(Type $type)
    {
        $this->type = $type;
    }

    public function getModel() 
    {
        return "This is a B model of type " . $this->type->getType();
    }
}

?>

File: Bridge Pattern/CS/Bmodel.cs
namespace BridgePattern
{
    public class Bmodel : Model
    {
        public Bmodel(IType type)
        {
            this.Type = type;
        }
        
        public override string ClassInfo()
        {
            return "B model of type " + this.Type.GetInfo();
        }
    }
}